### PR TITLE
Fix output dust warning

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: src/app/state/manager.rs [hunk 1]
```diff
@@ -839,6 +839,14 @@ impl ManagerSendOutput {
             ));
         }
 
+        if let Ok(address) = bitcoin::Address::from_str(&self.address.value) {
+            if amount <= address.script_pubkey().dust_value() {
+                return Err(Error::UnexpectedError(
+                    "Amount should be non-zero".to_string(),
```
Comment (edouardparis) [inline src/app/state/manager.rs:845]: need to change

File: src/app/view/manager.rs [hunk 2]
```diff
@@ -332,7 +332,7 @@ impl ManagerSendOutputView {
                     &amount,
                     RecipientMessage::AmountEdited,
                 )
-                .warning("Please enter a valid amount")
+                .warning("Please enter valid amount superior to dust limit")
```
Comment (darosior) [inline src/app/view/manager.rs:335]:
```suggestion
                .warning("Please enter a valid amount")
```
This gets messy

**Review by darosior — COMMENTED** [review]
Now the error always happen even when the field is empty. Was it the case before?
image

**Comment by edouardparis** [comment]
Self-ACK 13e0efbdb4c7634f79e3d1b70b0d133b37201954

**Pull request merged**